src/group/managed_group/test_managed_group.rs

```diff
@@ -0,0 +1,50 @@
+use crate::prelude::*;
```

Comment:
Why is this a unit test? Shouldn't all of this be usable from the outside?

Reply:
`PartialEq` is only derived for tests, we can't compare secrets from the outside.